src/sidebar/components/MentionSuggestionsPopover.tsx

```diff
@@ -0,0 +1,71 @@
+import { Popover } from '@hypothesis/frontend-shared';
+import type { PopoverProps } from '@hypothesis/frontend-shared/lib/components/feedback/Popover';
+import classnames from 'classnames';
+
+export type UserItem = {
+  username: string;
+  displayName: string | null;
+};
+
+export type MentionSuggestionsPopoverProps = Pick<
+  PopoverProps,
+  'open' | 'onClose' | 'anchorElementRef'
+> & {
+  /** List of suggestions to display */
+  suggestions: UserItem[];
```

Comment:
```suggestion
  users: UserItem[];
```

And same for other names. This makes it more obvious what kind of entity is being suggested. In future we might need to generalize this to inserting references to non-user items, but it is unclear whether that would use the same component or need a different one (as eg. the content would have a different layout).